### PR TITLE
Flush config for nifiscalcouncil

### PR DESCRIPTION
**Pull request by johangant**

### CODE REVIEW

File: config/nifiscalcouncil/config/core.entity_view_display.node.webform.teaser.yml [hunk 1]
```diff
@@ -18,18 +18,24 @@ bundle: webform
...
     region: content
   links:
-    weight: 100
     settings: {  }
     third_party_settings: {  }
+    weight: 100
     region: content
 hidden:
   langcode: true
+  search_api_excerpt: true
```
Comment (omahm) [inline config/nifiscalcouncil/config/core.entity_view_display.node.webform.teaser.yml:40]: new

File: config/nifiscalcouncil/config/core.entity_view_display.node.webform.teaser.yml [hunk 2]
```diff
@@ -18,18 +18,24 @@ bundle: webform
...
-    label: hidden
     type: text_summary_or_trimmed
-    weight: 101
+    label: hidden
     settings:
       trim_length: 600
     third_party_settings: {  }
+    weight: 101
+    region: content
+  content_moderation_control:
```
Comment (omahm) [inline config/nifiscalcouncil/config/core.entity_view_display.node.webform.teaser.yml:28]: new

File: config/nifiscalcouncil/config/core.entity_view_display.node.webform.default.yml [hunk 3]
```diff
@@ -18,23 +18,29 @@ bundle: webform
...
 content:
   body:
-    label: hidden
     type: text_default
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
     weight: 101
+    region: content
+  content_moderation_control:
```
Comment (omahm) [inline config/nifiscalcouncil/config/core.entity_view_display.node.webform.default.yml:27]: New but pretty much empty config

File: config/nifiscalcouncil/config/core.entity_view_display.node.webform.default.yml [hunk 4]
```diff
@@ -18,23 +18,29 @@ bundle: webform
...
+    type: webform_entity_reference_entity_view
     label: hidden
     settings: {  }
     third_party_settings: {  }
-    type: webform_entity_reference_entity_view
+    weight: 102
     region: content
 hidden:
   langcode: true
+  search_api_excerpt: true
```
Comment (omahm) [inline config/nifiscalcouncil/config/core.entity_view_display.node.webform.default.yml:46]: new

**Review by omahm — COMMENTED** [review]
LGTM. 
There is some additional config in there but it's either for stuff like hidden fields in the UI or an entry (IIRC, content moderation) but with blank config so it'll do sod all.

**Review by ooneill547 — APPROVED** [review]

**Pull request merged**